plugins/catalog-backend/src/next/database/DefaultProcessingDatabase.ts

```diff
@@ -123,6 +123,26 @@ export class DefaultProcessingDatabase implements ProcessingDatabase {
     );
   }
 
+  async updateProcessedEntityErrors(
+    txOpaque: Transaction,
+    options: UpdateProcessedEntityOptions,
+  ): Promise<void> {
+    const tx = txOpaque as Knex.Transaction;
+    const { id, errors } = options;
+
+    await tx<DbRefreshStateRow>('refresh_state')
+      .update({
+        errors,
+      })
+      .where('entity_id', id);
+
+    /*
+    if (refreshResult === 0) {
+      throw new NotFoundError(`Processing state not found for ${id}`);
+    }
+    */
```

Comment:
🔥 ?

Reply:
ah yes, missed to clean this up from a mistaken stash op